vectorsearch/params/aoss/10million/nmslib-cohere-768-dp.json

```diff
@@ -0,0 +1,28 @@
+{
+    "target_index_name": "target_index",
+    "target_field_name": "target_field",
+    "target_index_body": "indices/nmslib-index.json",
+    "target_index_dimension": 768,
+    "target_index_space_type": "innerproduct",
+    "id_field_name": "id",
+    
+    "target_index_bulk_size": 100,
+    "target_index_bulk_index_data_set_format": "hdf5",
+    "target_index_bulk_index_data_set_corpus": "cohere-10m",
+    "target_index_bulk_indexing_clients": 10,
+    
+    "hnsw_ef_search": 256,
+    "hnsw_ef_construction": 256,
+
+    "query_k": 100,
+    "query_body": {
+         "docvalue_fields" : ["id"],
```

Comment:
This is hardcoded but on line 7, it is configurable?

Reply:
param value is configurable. In this case, id is configured as field name , and, query body is configured as well. At this moment, there is no relation between two params.

Reply:
Im not sure I understand. Is it not possible to point docvalues_fields at "id" to avoid duplication?

Reply:
As discussed offline, we want to keep query body and id_field name to be independent. In future, if we see need to make doc_values field as common field, we can move this to procedure and remove reference.